invesalius/data/brainmesh_handler.py

```diff
@@ -7,32 +7,29 @@
 
 
 class Brain:
-    def __init__(self, img_path, mask_path, n_peels, affine_vtk):
+    def __init__(self, image, mask, n_peels, affine_vtk, window_width, window_level):
         # Create arrays to access the peel data and peel Actors
         self.peel = []
         self.peelActors = []
-        # Read the image
-        T1_reader = vtk.vtkNIFTIImageReader()
-        T1_reader.SetFileName(img_path)
-        T1_reader.Update()
+        self.window_width = window_width
+        self.window_level = window_level
+
         # Image
-        self.refImage = T1_reader.GetOutput()
-        # Read the mask
-        mask_reader = vtk.vtkNIFTIImageReader()
-        mask_reader.SetFileName(mask_path)
-        mask_reader.Update()
+        self.refImage = image
+
         # Use the mask to create isosurface
         mc = vtk.vtkContourFilter()
-        mc.SetInputConnection(mask_reader.GetOutputPort())
-        mc.SetValue(0, 1)
+        mc.SetInputData(mask)
+        mc.SetValue(0, 127)
         mc.Update()
 
         # Mask isosurface
         refSurface = mc.GetOutput()
+
         # Create a uniformly meshed surface
         tmpPeel = downsample(refSurface)
         # Standard space coordinates
-        mask_sFormMatrix = mask_reader.GetSFormMatrix()
+        mask_sFormMatrix = vtk.vtkMatrix4x4()
```

Comment:
because now the `mask` and `image` are already in the same space, we could remove any transformation. however, thinking that we would offer the user the possibility to load their own mask, it would be important to transform the mask from its space to the invesalius default